endpoints/IncomingEndpoint.ts

```diff
@@ -52,7 +52,7 @@ export class IncomingEndpoint extends ApiEndpoint {
 		const room = (await read.getRoomReader().getById(sessionId)) as ILivechatRoom;
 		switch (action) {
 			case EndpointActionNames.CLOSE_CHAT:
-				await closeChat(modify, read, sessionId);
+				await closeChat(modify, read, sessionId, this.app, CloseChatDescription.CLOSED_BY_VISITOR);
```

Comment:
This is confusing. Chat closed by the `/incoming` endpoint is not necessarily closed by the visitor.